include/libnuraft/log_entry.hxx

```diff
@@ -92,6 +103,14 @@ public:
         timestamp_us_ = t;
     }
 
+    uint32_t get_crc32() const {
+        return crc32_;
+    }
+
+    void set_crc32(uint32_t crc) {
+        crc32_ = crc;
+    }
```

Comment:
For backward compatibility (for the previous logs that do not have CRC, and they can be mixed with new logs with CRC), we should have one more field indicating if CRC exists or not, let's say `has_crc32_`. 

Otherwise, we cannot distinguish non-existing CRC and CRC=0. Also we need an API returning that flag: 
```c++
bool has_crc32() const () { ... }
```
that will be used in asio layer (replacing `crc32 != 0`).